Automattic/jetpack
# Social Menu & Icons: Remove Google Plus Support

Partially fixes [Automattic/wp-calypso#31299 (comment) ](https://github.com/Automattic/wp-calypso/issues/31299#issuecomment-470974429) 

#### Changes proposed in this Pull Request:


Removes Google+ support from the Social Menu and Social Icons, because imo it seems better that current usages should fallback to the Google icon instead of a service which is shut down (by the next release). 

#### Testing instructions:





* Go to the Customizer with a theme that supports them 
* Add Google+ to the social menu and the social icons widget 
* Google logo appear? All others working fine?

I suppose the real question here is that if this should even be merged, what should be displayed for current usages? I think the Google logo is the best and easiest option personally, but would appreciate feedback on that. 

#### Proposed changelog entry for your changes:


* Social Menus & Icons: Remove Google+ support.
